src/components/CheckboxInput/CheckboxInput.tsx

```diff
@@ -0,0 +1,38 @@
+﻿import React from 'react';
+import CheckboxIcon from '../../icons/CheckboxIcon';
+import { useFilters } from '../../context/FiltersContext';
+import { InputProps } from '../../types/types';
+
+const CheckboxInput = ({ name, label, group, type }: InputProps) => {
+   const { filters, setFilters } = useFilters();
```

Comment:
**issue:** You wired app specific logic with dumb components. In current implementation it is impossible to reuse this Checkbox in other part of app which is not filter related or just other app. Such components like Checkboxes, Inputs, Buttons should be independent pieces of code. Think of it like it was a design systejm